schedule/yast/maintenance/yast-mru-install-minimal-with-addons.yaml

```diff
@@ -0,0 +1,31 @@
+---
+name: yast-mru-install_dev
+vars:
+  DUD_ADDONS: sdk
+  PATTERNS: base,minimal,apparmor
+  YUI_REST_API: 1
+schedule:
+  - installation/bootloader_start
+  - installation/setup_libyui
+  - installation/product_selection/install_SLES
+  - installation/licensing/accept_license
+  - installation/registration/register_via_scc
+  - installation/module_registration/register_extensions_and_modules
+  - installation/add_on_product_installation/add_additional_products
+  - installation/add_on_product/skip_install_addons
+  - installation/system_role/accept_selected_role_SLES_with_GNOME
+  - installation/partitioning/accept_proposed_layout
+  - installation/clock_and_timezone/accept_timezone_configuration
+  - installation/authentication/use_same_password_for_root
+  - installation/authentication/default_user_simple_pwd
+  - installation/select_patterns
+  - installation/installation_settings/validate_default_target
+  - installation/bootloader_settings/disable_boot_menu_timeout
+  - installation/launch_installation
+  - installation/confirm_installation
+  - installation/performing_installation/perform_installation
+  - installation/logs_from_installation_system
+  - installation/performing_installation/confirm_reboot
+  - installation/grub_test
```

Comment:
[this](https://openqa.suse.de/tests/9719688#step/reconnect_mgmt_console/1) should not work with this yaml, could you double check that we have all the yaml needed for `yast-mru-install-minimal-with-addons`

Reply:
I have verified with this yaml file: https://openqa.suse.de/tests/9715588# It can work.

Reply:
you should put in WIP the PR when doing this changes commenting, otherwise could be accidentally merged.